### DISCUSSION TITLE
Re-Enroll Heavy Node

### DISCUSSION THREAD

**Discussion by Mfisher925** [Security-Onion-Solutions/securityonion#3632] [Unsupported Versions]
Recently an update went awry on our Manager node in a distributed deployment.

We reinstalled the Manager node with all same settings and creds as the previous.

Everything on the Manager seems to be working correctly, however the Heavy nodes we previously had on our grid are not present on the Manager's grid.

Is there a command/setup that needs to be run to re-enroll the Heavy nodes back to this new Manager?

**Comment by weslambert** [comment]
Is it just they they aren't showing in the Grid view? Are they reachable via Salt, via `salt "*" test.ping`?

**Reply by Mfisher925** [reply]
I never did try that so I couldn't tell you, thank you though.  We ended up re-installing the newest iso for the Heavy Nodes from scratch.
